3V0L/bootcamp-c2
# Tests and functionalities for users and books

**What does this PR do?**
This contains the tests for the API as well as the functions to make the tests pass.
The API endpoints are:
- POST  /api/books
-PUT /api/books/<bookId>
-DELETE /api//books/<bookId>
-GET  /api/books
-GET  /api/books/<bookId>
-POST  /api/users/books/<bookId>
-POST /api/auth/register
-POST /api/auth/login
-POST /api/auth/logout
-POST /api/auth/reset-password
-POST /api/auth/change-password

**How should this be manually tested?**
Clone repository and install requirements. After that run pytest to ensure all tests are still passing.

**What are the relevant pivotal tracker stories?**
https://www.pivotaltracker.com/story/show/156252444
https://www.pivotaltracker.com/story/show/156252459
https://www.pivotaltracker.com/story/show/156307210
https://www.pivotaltracker.com/story/show/156307326

Screenshots (if appropriate)

![screenshot 17](https://user-images.githubusercontent.com/33119403/38177099-c3991682-3603-11e8-97c8-4d693dc63e2f.png)
